CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## v0.0.2 [2022-07-21]
+
+_Bug fixes_
+
+- Fixed the brand color of the social graphic icon. ([#4](https://github.com/turbot/steampipe-plugin-trivy/pull/4))
```

Comment:
```suggestion
- Fixed the brand color. ([#4](https://github.com/turbot/steampipe-plugin-trivy/pull/4))
```

The social graphic image was not affected by this, just the brand color used when displaying the icon on the Hub site (potentially other places).